src/stores/Object.js

```diff
@@ -685,6 +788,45 @@ class ObjectStore {
       message: "Successfully deleted non-owner cap"
     });
   });
+
+  @action.bound
+  GetContentObjectMetadata = flow(function * ({
+    libraryId,
+    objectId,
+    versionHash,
+    writeToken,
+    metadataSubtree="/"
+  }) {
+    return yield Fabric.GetContentObjectMetadata({
+      libraryId,
+      objectId,
+      versionHash,
+      writeToken,
+      metadataSubtree
+    });
+  });
+
+  @action.bound
+  LatestVersionHash = flow(function * ({objectId, versionHash}) {
+    if(!objectId) objectId = Fabric.utils.DecodeVersionHash(versionHash).objectId;
+
+    return yield Fabric.LatestVersionHash({
+      objectId,
+      versionHash
+    });
+  });
+
+  @action.bound
+  GetIndexerSetting = flow(function * ({typeId}) {
```

Comment:
this is actually probably already present in `object.typeInfo`. That's where it determines manage app url, which is also in type meta.

Reply:
Ah you're right